### PR TITLE
Update to NuGet 3.2 RTM

### PR DESCRIPTION
**Pull request by rainersigwald**

### CODE REVIEW

File: dir.targets [hunk 1]
```diff
@@ -95,7 +95,7 @@
 
     <!-- Download latest nuget.exe -->
     <DownloadFile FileName="$(NuGetToolPath)"
-                  Address="https://dist.nuget.org/win-x86-commandline/v3.2.0-rc/nuget.exe"
+                  Address="https://dist.nuget.org/win-x86-commandline/v3.2.0/nuget.exe"
```
Comment (rainersigwald) [inline dir.targets]: Question:  should we use `/latest/`?

Comment (dsplaisted) [inline dir.targets]: I think either way would be OK.  Using a specific version means it's more likely we'll get the same behavior if we try to build at a previous point in the commit history.  However, using the latest version means we don't have to update it, and is the pattern that was used for NuGet v2.

**Comment by dsplaisted** [comment]
LGTM.

File: dir.targets (hunk 1 above)
Comment (rainersigwald) [inline dir.targets]: Ok, I'll go with the explicit 3.2 for now and we'll see if the churn gets too annoying.

**Comment by AndyGerlicher** [comment]
:shipit:

**Pull request merged**